packages/client/components/Dashboard/DashSidebar.tsx

```diff
@@ -57,8 +55,78 @@ const Wrapper = styled('div')({
   flexDirection: 'column'
 })
 
+const OrgName = styled('div')({
+  paddingTop: 8,
+  paddingLeft: 8,
+  fontWeight: 600,
+  fontSize: 12,
+  lineHeight: '24px',
+  color: PALETTE.SLATE_500,
+  [makeMinWidthMediaQuery(Breakpoint.SIDEBAR_LEFT)]: {
+    paddingLeft: 16
+  }
+})
+
+interface Props {
+  isOpen: boolean
+  viewerRef: DashSidebar_viewer$key | null
+}
+
 const DashSidebar = (props: Props) => {
-  const {isOpen, viewer} = props
+  const {isOpen, viewerRef} = props
+  const location = useLocation()
+  const {pathname} = location
+
+  const viewer = useFragment(
+    graphql`
+      fragment DashSidebar_viewer on User {
+        ...StandardHub_viewer
+        ...DashNavList_viewer
+        featureFlags {
+          checkoutFlow
+        }
+        organizations {
+          id
+          name
+        }
+      }
+    `,
+    viewerRef
+  )
+
+  if (!viewer) return null
+  const {featureFlags, organizations} = viewer
+  const showOrgSidebar = featureFlags.checkoutFlow && pathname.startsWith(`/me/organizations`)
```

Comment:
-1 instead of `pathname.startsWith` we should use
```ts
const orgRouteMatch = useMatch<{orgId: string}>('/me/organizations/:orgId')
const showOrgSidebar = featureFlags.checkoutFlow && orgRouteMatch
```
You have to check if you need to set `strict` or `exact` in addition.